wagtail/wagtailadmin/views/chooser.py

```diff
@@ -22,25 +22,29 @@ def get_querystring(request):
 def browse(request, parent_page_id=None):
     ITEMS_PER_PAGE = 25
 
-    page_type = request.GET.get('page_type') or 'wagtailcore.page'
-    content_type_app_name, content_type_model_name = page_type.split('.')
+    page_types = request.GET.get('page_types', 'wagtailcore.page').split(',')
 
-    try:
-        content_type = ContentType.objects.get_by_natural_key(content_type_app_name, content_type_model_name)
-    except ContentType.DoesNotExist:
-        raise Http404
-    desired_class = content_type.model_class()
+    desired_classes = []
+    for page_type in page_types:
+        content_type_app_name, content_type_model_name = page_type.split('.')
```

Comment:
This will throw a ValueError if `page_type` does not contain exactly one dot. Consider wrapping it in a `try: except ValueError:`. Alternately, consider using [`resolve_model_string`](https://github.com/torchbox/wagtail/blob/00f707a4389403ac9f0a0769d3afd8c71f4966c6/wagtail/wagtailcore/utils.py#L16) which takes care of this for you.